lib/probes/http.js

```diff
@@ -53,34 +75,40 @@ function patchClient (module, protocol) {
       span.async = true
 
       // Parse options object
-      if (typeof options === 'string') {
-        options = url.parse(options)
+      /*
+      if (typeof urlstring === 'string') {
+        urlopts = url.parse(urlstring)
       }
       const parsed = extend({}, options)
+      // */
```

Comment:
this commented out code can probably go because it is on line 49.. now
It seams a bit out of context here.